### PR TITLE
Face mask detection using python #48

### LINKED ISSUE #48: face mask detection detection
# Description

 Detects if a person is wearing a mask primarily in public places or not.

## Type of issue
- [X] Feature (New Script)
- [ ] Bug
- [ ] Documentation

## Checklist:

- [X] I have read the project guidelines.
- [X] I have checked all the existing projects, before submitting a new project issue.
- [X] I have checked previous issues to avoid duplicates.
- [X] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by hacky1997**
# Description

Face Mask Detection system built with OpenCV, Keras/TensorFlow using Deep Learning and Computer Vision concepts in order to detect face masks in static images as well as in real-time video streams.

Fixes #48 

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [X] I have named my files and folder, according to this project's guidelines.
- [X] My code follows the style guidelines of this project.
- [X] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [X] I have commented on my code, particularly in hard-to-understand areas.
- [X] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [X] I have included a requirements.txt file (if external libraries are required.)
- [X] My changes do not produce any warnings.
- [X] I have added a working sample/screenshot of the script.

### CODE REVIEW

**Comment by chavarera** [comment]
@hacky1997 please don't upload dataset as it is compress it and upload

**Comment by AdityaJ7** [comment]
@hacky1997  Please don't include all the images, just add a working example of the script via screenshot  and add a couple of images that u used

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
perform the requested changes!

**Review by chavarera — CHANGES_REQUESTED** [review]
add dataset in zip,tar,gzip compressed format

**Comment by hacky1997** [comment]
@chavarera & @AdityaJ7 My network is slow right now. I will do this a bit later for sure.

**Comment by chavarera** [comment]
> @chavarera & @AdityaJ7 My network is slow right now. I will do this a bit later for sure.

are you working on it ?

**Comment by hacky1997** [comment]
@chavarera Yes! when I have compressed the dataset to tar file it takes around 125 MB. So, it's the problem which I am facing currently. Could you please help me to sort it out?

**Comment by AdityaJ7** [comment]
> @chavarera Yes! when I have compressed the dataset to tar file it takes around 125 MB. So, it's the problem which I am facing currently. Could you please help me to sort it out?

oKAY @hacky1997 do one thing remove all these images and add the source from which these files can be attained in the README.md file

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
oKAY @hacky1997 do one thing remove all these images and add the source from which these files can be attained in the README.md file

Your final folder must contain -

1.A Readme file with a better description of the script and how to use it and the link to the place from where these images can be downloaded
2.A screenshot of the script in action
3.A requirements.txt file containing external libraries used in the script
4.Your scripts

**Comment by AdityaJ7** [comment]
will close this pr after 3 days @hacky1997
